kickstarts/partials/post/db_init.ks.erb

```diff
@@ -4,7 +4,7 @@ cat > /bin/appliance-initialize.sh <<EOF
 [[ -d /var/opt/rh/rh-postgresql95/lib/pgsql/data/base ]] && exit 0
 [[ -s /etc/default/evm ]] && source /etc/default/evm
 echo "Initializing Appliance, please wait ..." > /dev/tty1
-appliance_console_cli --region 0 --internal --password smartvm
+appliance_console_cli --region 0 --internal --password smartvm --key
```

Comment:
This isn't run during the kickstart. We print the file here (I'm not sure why) and it runs on first boot.

This should really be moved out with the rest of our systemd unit files.